delegator/src/contract/dto/contract.dto.ts

```diff
@@ -3,4 +3,7 @@ import { ApiProperty } from '@nestjs/swagger'
 export class ContractDto {
   @ApiProperty()
   address: string
+
+  @ApiProperty()
+  allowAllFunctions: boolean
```

Comment:
Do we still use it? I think you removed it from the validation function.

Reply:
Yes. I decided to remove logic of `allowAllFunctions`. 
I removed all with latest changes.